src/services/updateRoleServices.js

```diff
@@ -0,0 +1,24 @@
+const User = require('../database/models').User;
+
+
+class UpdateRoleServices {
+   static async updateRole(data) {
+    const user = await User.findOne({ where: { id: data.id } });    
+    if (!user) {
+      return false;
+    } else {
+        await User.update(
+          { role: data.role },
+          {
+            where: {
+              id: user.id,
+              role: user.role,
+            },
+          }
+        );
+        return true;
+    }
+  }
+}
```

Comment:
- Service should only hold the query. Ask Yves for help
- This is user related service